src/react/DeviceStatusPage/AddToGroupModal.tsx

```diff
@@ -19,6 +19,8 @@
 import React, { useState } from 'react';
 import { Button, Modal, Spinner } from 'react-bootstrap';
 
+import Icon from '../components/Icon';
```

Comment:
we should use a webpack alias for components. Expecially the "global" ones

Reply:
I agree, indeed we can certainly think about a dedicated PR for that.